ssstar/src/objstore/s3.rs

```diff
@@ -582,6 +584,75 @@ impl S3Bucket {
             key
         }
     }
+
+    /// Compute what the multipart chunk size should be, based on the user's configured chunk size
+    /// but also informed by the estimated size of the object
+    fn compute_multipart_chunk_size(
```

Comment:
In the code I see why an `Option` was used, but the explanation should be also in the doc comment.

Reply:
Added comment to explain